ArduPlane/control_modes.cpp

```diff
@@ -159,34 +159,60 @@ void Plane::reset_control_switch()
 /*
   called when entering autotune
  */
-void Plane::autotune_start(void)
+void Plane::autotune_start(uint8_t tune_type)
 {
     gcs().send_text(MAV_SEVERITY_INFO, "Started autotune");
-    rollController.autotune_start();
-    pitchController.autotune_start();
-    yawController.autotune_start();
+    switch (tune_type) {
```

Comment:
Shouldn't be a switch - this is a bitmask.

Did I say something wrong on my previous review, I wonder?

Reply:
@peterbarker,  I have only one doubt here. that is `AP_AutoTune::ATType` members having 0 1 and 2 values. should'nt we pass those directly to this from mavlink.cpp. And using switch we just execute that axis.